exporter/elasticsearchexporter/internal/serializer/otelserializer/serializeprofiles/model.go

```diff
@@ -103,7 +140,7 @@ type ExeMetadataParams struct {
 	LastSeen   uint32 `json:"timestamp"`
 	BuildID    string `json:"buildid"`
 	FileName   string `json:"filename"`
-	EcsVersion string `json:"ecsversion"`
+	EcsVersion string `json:"ecs.version"`
```

Comment:
Just to double check the expectations here, are you expecting the output to be:
```json
{ "ecs": {"version": val}}
```

Or 
```json
{"ecs.version": val}
```
?